README.md

```diff
@@ -8,6 +8,24 @@ To learn how to install and use this connector please visit our [support page](h
 ## Sample report
 A Power BI report template has been included with [each release](https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/releases). After installing the connector you can download and open this report in Power BI desktop. You will be prompted to enter an API key which you can generate by visiting [API Tokens](https://app.safetyculture.com/account/api-tokens) page in SafetyCulture website.
 
+# Frequently Asked Questions (FAQ)
+## What’s the purpose of the “Org Name”?
+The “Org Name” helps distinguish between organizations locally. This can be particularly useful if you choose to consume data from multiple iAuditor organizations. The “Org Name” information doesn’t get transmitted to our servers and is only stored locally on your machine.
+
+You can label the “Org Name” freely, as long as it's unique.
+
+## Can the data be filtered when loading data for the first time?
+Yes, the “inspections” and “inspection_items” data sets can be filtered using the provided functions of “GetInspections” and “GetInspectionItems”, respectively. Both can be found in the [nav table.](./iAuditor.pq#L274-L286)
```

Comment:
I'm a bit worried about including line number (`./iAuditor.pq#L274-L286`) here. Can get out of date quickly.

Reply:
Any proposals for this..? Does github support anchoring of any sort?

Reply:
I'm not sure if it does. But I don't see any point referring to the code. You can get rid of the last sentence.

Reply:
I think its OK for now, at least it links somewhere.

Reply:
My understanding is this FAQ is more targeted towards the end consumer. They just use the connector as is and never bother with the code.
When we say `GetInspections` and `GetInspectionItems` functions. It's clear that we are referring to the functions that they see when connect to iAuditor in Power BI.
Having those code line numbers will only confuse these customers.

Reply:
That's a very good point, I was thinking of aiming this section both for developers as well as plug and play users that are a little curious. Removed nonetheless.